lib/chainpoint.rb

```diff
@@ -55,78 +57,66 @@ def self.submit(hash)
   # Submit a hash to a chainpoint server
   #
   # @param data [String] hash String which you want to submit into the chainpoint server.
-  # @return [Array] An array of hashes containing the keys `hash`, `hash_id_node` and `uri`
+  # @return [Array] An array of Chainpoint::Hash objects
   def submit(hash)
-    uri = URI(@server_url + '/hashes')
-    request = Net::HTTP::Post.new(uri, 'Content-Type' => 'application/json')
-    request.body = { hashes: [hash] }.to_json
-    response = Net::HTTP.start(uri.hostname, uri.port, use_ssl: uri.scheme == 'https') do |http|
-      http.request(request)
-    end
+    @server_urls.map do |url|
+      uri = URI(url + '/hashes')
+      request = Net::HTTP::Post.new(uri, 'Content-Type' => 'application/json')
+      request.body = { hashes: [hash] }.to_json
 
-    hashes = JSON.parse(response.body)['hashes']
-    hashes.map { |hash| hash.merge('uri' => @server_url) }
+      response = Net::HTTP.start(uri.hostname, uri.port, use_ssl: uri.scheme == 'https') do |http|
+        http.request(request)
+      end
+
+      JSON.parse(response.body)['hashes'].map { |hash| hash.merge('uri' => url) }
+    end.flatten
   end
 
   # Get proof data from the chainpoint server.
   #
-  # @param hash_id_node [String] hash id node of the proof
-  def self.get_proof(hash_id_node)
-    self.new().get_proof(hash_id_node)
+  # @param URI [String] hash id node of the proof
+  # @param node_hash_id [String] hash id node of the proof
+  # @return [JSON]
+  def self.get_proof(uri, node_hash_id)
```

Comment:
@jaredmoody 

I have a question.
I created this function before. Now we can use `Chainpoint` as `Chainpoint.get_proof` and `Chainpoint.new.get_proof`.
Is it useful? What do you think?

Reply:
I think it can be nice to include shortcuts like that. I included this one to follow the pattern you had created.